### ISSUE TITLE
about dataset

### ISSUE THREAD

**Issue by missqi18** [stefan-ainetter/grasp_det_seg_cnn#3]
hello, I have read your paper, and in your experiment section, there are only the results for grasp accuracy and semantic segmentation (namely Deg_Seg) for the OCID_grasp dataset, so if the code here only can obtain the result of Deg_Seg? 
And how to use  the Jacquard dataset in your algorithm？

Looking forward to your reply!

**Comment by stefan-ainetter** [comment]
hi @missqi18,

The train and test scripts provided in this repository are specifically for training/testing Det_Seg (Grasp Detection Branch and Segmentation Branch) with the OCID_grasp dataset.

However, as the Grasp Detection Branch and the Segmentation Branch can be used independently, you can simply remove the Segmentation Branch and use only the Grasp Detection Branch. We did this for our experiments in Section VI for the Cornell grasp dataset and Jacquard dataset ( see results for Detection(ours) in the paper). Of course, you can also train only the Grasp Detection Branch with OCID_grasp by removing the Segmentation Branch.

Currently, this repository does not contain the code to train/test our method on the Jacquard dataset. We may add this in the future. In the meantime, if you want to integrate the Jacquard dataset by yourself, you have to do the following modifications:
1) Download the Jacquard dataset.
2) Modifiy the train/test scripts
3) Add a dataloader for the Jacquard dataset (similar to the one used for OCID_grasp, see grasp_det_seg/data_OCID).
